custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -189,7 +189,8 @@ def get_payload(self, repeat_record, person_case):
             "HIVTestDate": _format_date(person_case_properties, 'hiv_test_date'),
             "CPTDeliverDate": _format_date(person_case_properties, 'cpt_initiation_date'),
             "ARTCentreDate": _format_date(person_case_properties, 'art_initiation_date'),
-            "InitiatedOnART": art_initiated.get(person_case_properties.get('art_initiated', 'no')),
+            "InitiatedOnART": art_initiated.get(
+                person_case_properties.get('art_initiated', 'no'), art_initiated['no']),
```

Comment:
@snopoke This is added here for empty string but yes would take care of any bad values as well.